app/src/main/java/com/eveningoutpost/dexdrip/Models/UserError.java

```diff
@@ -151,52 +155,139 @@ public static void e(String a, String b){
             new UserError(a, b);
         }
 
-        public static void e(String a, String b, Exception e){
-            android.util.Log.e(a, b, e);
-            new UserError(a, b + "\n" + e.toString());
+        public static void e(String tag, String b, Exception e){
+            android.util.Log.e(tag, b, e);
+            new UserError(tag, b + "\n" + e.toString());
```

Comment:
This and similar lines, is this change from `a` to `tag` just a refactor to make it more readable?